### PR TITLE
Create CODE_OF_CONDUCT.md

### PR DESCRIPTION
**Pull request by tharunc**
CODE_OF_CONDUCT.md has been created.

### CODE REVIEW

**Comment by arpansac** [comment]
Hey I went through it, just 2 things, it
- It needs to mention that this code of conduct is for the developers contributing to Commudle
- It should mention the links to the privacy policy and T&C which are present on the official website
- It's link should be mentioned in the readme

**Review by arpansac — CHANGES_REQUESTED** [review]
Hey I went through it, just these things, it

It needs to mention that this code of conduct is for the developers contributing to Commudle
It should mention the links to the privacy policy and T&C which are present on the official website
It's link should be mentioned in the readme

**Comment by tharunc** [comment]
Sure, but can you please mention the Line for the formatting to be done.

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Contributor Covenant Code of Conduct
```
Comment (arpansac) [inline CODE_OF_CONDUCT.md]: For Commudle

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+reported by contacting the project team. All
+complaints will be reviewed and investigated and will result in a response that
+is deemed necessary and appropriate to the circumstances. The project team is
+obligated to maintain confidentiality with regard to the reporter of an incident.
+Further details of specific enforcement policies may be posted separately.
+
+Project maintainers who do not follow or enforce the Code of Conduct in good
+faith may face temporary or permanent repercussions as determined by other
+members of the project's leadership.
+
```
Comment (arpansac) [inline CODE_OF_CONDUCT.md:67]: Add a section for T&C, Privacy Policy

**Review by arpansac — COMMENTED** [review]
> Sure, but can you please mention the Line for the formatting to be done.

Sure, added 2 comments, for the 3rd point please put it just below the contribution guidelines link in the Readme

**Comment by arpansac** [comment]
Cool, will review and merge by tomorrow

**Comment by tharunc** [comment]
Kindly add LEVEL tag for this merge.

**Pull request merged**